modules/ocf_hpc/manifests/compute.pp

```diff
@@ -0,0 +1,35 @@
+class ocf_hpc::compute {
+  require ocf_hpc
+
+  # install proprietary nvidia drivers
+  package { ['nvidia-driver', 'nvidia-settings', 'nvidia-cuda-toolkit']:; }
+
+  file { '/etc/slurm-llnl/gres.conf':
+    content => template('ocf_hpc/gres.conf.erb'),
+    mode    => '0644',
+    owner   => 'slurm',
+    group   => 'slurm',
+  }
+  # Kernel option to enable memory as a consumable gres resource
+  augeas { 'grub':
+    incl        => '/etc/default/grub',
+    lens        => 'ShellVars.lns',
+    changes     => [
+      "set GRUB_CMDLINE_LINUX '\"cgroup_enable=memory swapaccount=1\"'",
+    ],
+    refreshonly => true
+  } ~> exec { 'update-grub':
+    user => 'root',
```

Comment:
You'll still want to set `refreshonly => true`

Reply:
Ah I set it in augeas for some reason, mb